src/zjs_modules.c

```diff
@@ -227,16 +230,23 @@ static ZJS_DECL_FUNC(native_print_handler)
     zjs_free(str);
     return ZJS_UNDEFINED;
 }
-
-static ZJS_DECL_FUNC(stop_js_handler)
+static void zjs_stop_js()
 {
-#ifdef CONFIG_BOARD_ARDUINO_101
-#ifdef CONFIG_IPM
-    zjs_ipm_free_callbacks();
-#endif
-#endif
     zjs_modules_cleanup();
+    zjs_remove_all_callbacks();
+    #ifdef CONFIG_BOARD_ARDUINO_101
+    #ifdef CONFIG_IPM
+        zjs_ipm_free_callbacks();
+    #endif
+    #endif
```

Comment:
Keep the preprocessor commands at the beginning of the line, this is what we do everywhere. The function call shouldn't be indented further, just the same as the other statements. Probably this was just an accident?